### PR TITLE
trying to add user schema

### PR DESCRIPTION
**Pull request by ericshuanders**
Began userSchema

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const app = express();
 app.use(express.json());
 
 // Unauthenticated routes
-app.use(openRoutes);
```
Comment (Stommy123) [inline server/app.js:12]: I don't think we want the trailing `/`. Let's say this is one of your open routes
```
app.get('/login', loginHandler)
```

The path will actually be `/api//login`

**Review by Stommy123 — APPROVED** [review]
Looks good! Just one nit with the path

File: server/db/models/user.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                required: true,
+                trim: true
+            },
+            lastName:{
+                type: String,
+                required: true,
+                trim: true
+            }
+        }
+    ],
```
Comment (Stommy123) [inline server/db/models/user.js:20]: Actually i didn't catch this sorry. I think this shouldn't be an array, but rather the `firstName` and `lastName` property should just exist on the root user obj

**Review by Stommy123 — CHANGES_REQUESTED** [review]
Hey, sorry changing my approval to request changes for the `name` array

**Review by Stommy123 — APPROVED** [review]
Awesome, thanks for making the changes!

**Review by saritasalazar — APPROVED** [review]

**Pull request merged**